k9mail/src/main/res/values/pep_strings.xml

```diff
@@ -353,8 +353,8 @@
     <string name="compose_message_too_many_attachments">Cannot add more than %1$d attachments.</string>
 
     <plurals name="compose_unsecure_delivery_warning">
-        <item quantity="one">%1$d unsecure recipient. Tap here to remove it.</item>
-        <item quantity="other">%1$d unsecure recipients. Tap here to remove them.</item>
+        <item quantity="one">%1$d unknown recipient.</item>
```

Comment:
ok we changed from unknown to unsecure, but this is in general, so was it also a requirement in the same ticket?

Reply:
Yes this is the warning that appears when composing a message to unknown recipients (we have no key for). It was suggested by Lisa Li the UX and it fits much better with the use we are giving now to this functionality, until something is put in place to take into account other kinds of unsecure and mistrusted recipients.